source/devel_docker_ci.rst

```diff
@@ -0,0 +1,251 @@
+=============================
+Docker CI with GitHub Actions
+=============================
+
```

Comment:
I think we should add an introductory paragraph here. E.g.:

All docker containers, and all code should be tested. Continuous Integration (CI) with GitHub Actions takes care of this for repositories that has a Docker container. Repositories that comes without a Docker container can be tested with, e.g., Travis CI and Coveralls (for monitoring test coverage). 

(I feel it is a bit overkill to create a container for unit testing in, e.g., MMD, so it seems Travis is better for that. However, perhaps Docker CI would be useful to solve the automatic building of pdfs in MMD as requested in https://app.zenhub.com/workspaces/s-enda-5e7b1b4acfc5ca185fb6b53c/issues/steingod/mmd/4? But this anyway means that we need to commit and push the result, though.. Is there a good way to do that?)

Reply:
I agree. It's possible to update current repository with changed files on build, or maybe release. Ref https://github.community/t5/GitHub-Actions/Can-GitHub-actions-directly-edit-files-in-a-repository/td-p/50551

Need to be tested.

Reply:
Added your ingress to the documentation.